test/board_test.rb

```diff
@@ -27,4 +29,51 @@ def test_it_can_validate_coordinates
     refute @board.valid_coordinate?("E1")
     refute @board.valid_coordinate?("A22")
   end
+
+  def test_it_coordinate_length_equal_ship_length
+    refute @board.valid_placement?(@cruiser, ["A1", "A2"])
+    assert @board.valid_placement?(@cruiser, ["A1", "A2", "A3"])
+    assert @board.valid_placement?(@submarine, ["B1", "B2"])
+    refute @board.valid_placement?(@submarine, ["A2", "A3", "A4"])
+  end
+
+  def test_it_finds_consecutive_numbers
+    refute @board.numbers_consecutive?(["A1", "A2", "A4"])
+    assert @board.numbers_consecutive?(["A2", "A3", "A4"])
+    refute @board.numbers_consecutive?(["B1", "B3"])
+    assert @board.numbers_consecutive?(["B1", "B2"])
+    refute @board.numbers_consecutive?(["A3", "A2", "A1"])
+  end
+
+  def test_it_finds_consecutive_letters
+    refute @board.letters_consecutive?(["A1", "B1", "D1"])
+    assert @board.letters_consecutive?(["A1", "B1", "C1"])
+    refute @board.letters_consecutive?(["B1", "A1"])
+    assert @board.letters_consecutive?(["C1", "D1"])
+  end
+
+  def test_it_finds_consecutive_coordinates
+    refute @board.valid_placement?(@cruiser, ["A1", "A2", "A4"])
+    assert @board.valid_placement?(@cruiser, ["A2", "A3", "A4"])
+    refute @board.valid_placement?(@submarine, ["B1", "B3"])
```

Comment:
Ask about commenting out past tests that work to shorten length of code and potentially increase run speed.